build_tools/kokoro/gcp_ubuntu/cmake/linux/x86-turing/build.sh

```diff
@@ -57,15 +57,24 @@ CMAKE_ARGS=(
   "-DIREE_BUILD_PYTHON_BINDINGS=ON"
 
   "-DIREE_ENABLE_ASSERTIONS=ON"
+  " -DIREE_BUILD_TESTS=ON"
 
   # Enable CUDA backend to test on Turing hardware.
   "-DIREE_TARGET_BACKEND_CUDA=ON"
   "-DIREE_HAL_DRIVER_CUDA=ON"
 )
 
+echo "Configuring CMake"
 "$CMAKE_BIN" "${CMAKE_ARGS[@]?}" "$@" ..
+
+echo "Building all"
+echo "------------"
 "$CMAKE_BIN" --build . -- -k 0
 
+echo "Building test deps"
+echo "------------------"
+"$CMAKE_BIN" --build . --target iree-test-deps -- -k 0
```

Comment:
Note that since this is a post-submit build by default you'll need to trigger a build manually if you want to test it on presubmit.

Reply:
With these latest changes, I've verified fixes to `iree/gcp_ubuntu/cmake/linux/x86-turing/main[default]` on my own machine. Applied the same fix to the android build. This should solve the issue.